server/conf/application.conf

```diff
@@ -78,6 +78,10 @@ auth {
   oidc_override_logout_url = ${?APPLICANT_OIDC_OVERRIDE_LOGOUT_URL}
   oidc_post_logout_param = ${?APPLICANT_OIDC_POST_LOGOUT_REDIRECT_PARAM}
   oidc_logout_client_id_param = ${?APPLICANT_OIDC_LOGOUT_CLIENT_PARAM}
+
+  # Direct URI to create new account.
+  register_uri = ${?APPLICANT_REGISTER_URI}
+  register_uri = ${?IDCS_REGISTER_URI}
```

Comment:
NB: I'm slightly concern having two different environment variables setting the same value could make troubleshooting difficult. Is there anything we can do to log an error if both APPLICANT_REGISTER_URI and IDCS_REGISTER_URI are set? If both are set is that such an exceptional event we should prevent the application from starting?

Reply:
As I understand it we would want to have Seattle move to the new general APPLICANT one then remove the IDCS one completely.

Is that correctly @azizova-leyla ?

Reply:
I switched Seattle over

Reply:
that was fast =) https://github.com/civiform/civiform/pull/3800 has cleanup.

Reply:
Just wanna get it done